testsuite/domain/pom.xml

```diff
@@ -341,8 +341,6 @@
                                     <argLine>-Djava.util.logging.manager=org.jboss.logmanager.LogManager ${jvm.args.ip.client} ${jvm.args.timeouts} ${surefire.system.args}</argLine>
                                     
                                     <systemPropertyVariables>
-                                        <server.jvm.args>-Dmaven.repo.local=${settings.localRepository}</server.jvm.args>
```

Comment:
Note that a few lines below what the GH diff UI shows is another line that configures this same property. So this one is cruft.